src/utils/intl.js

```diff
@@ -0,0 +1,22 @@
+export let { ListFormat } = Intl;
+
+// Simple polyfill for Intl.ListFormat for en-US.
+//
+// Remove once more browsers support it:
+// https://caniuse.com/mdn-javascript_builtins_intl_listformat
+if (!ListFormat) {
+	ListFormat = class ListFormat { // eslint-disable-line no-shadow
```

Comment:
Correct me if I'm wrong, but the 'en-US' argument will get ignored here?

Reply:
Yes, we ignore the locale string. This is fine since the whole website is in American English anyway. Hence the comment:

```js
// Simple polyfill for Intl.ListFormat for en-US.
```